bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -195,30 +196,30 @@ public boolean equals(Object obj) {
   private SerializableScan serializableScan;
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
    * @param projectId The project ID for the cluster.
-   * @param zone The zone where the cluster is located.
-   * @param cluster The cluster ID for the cluster.
-   * @param table The table to connect to in the cluster.
+   * @param zoneId The zone where the cluster is located.
+   * @param clusterId The cluster ID for the cluster.
+   * @param tableId The table to connect to in the cluster.
    * @param scan The {@link Scan} that will be used to filter the table.
    */
-  public CloudBigtableScanConfiguration(String projectId, String zone, String cluster,
-      String table, Scan scan) {
-    this(projectId, zone, cluster, table, scan, Collections.<String, String> emptyMap());
+  public CloudBigtableScanConfiguration(String projectId, String zoneId, String clusterId,
+      String tableId, Scan scan) {
+    this(projectId, zoneId, clusterId, tableId, scan, Collections.<String, String> emptyMap());
   }
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
```

Comment:
"Creates a {@link CloudBigtableScanConfiguration} that will perform the specified scan in the table, with additional configuration options."